src/ts/component/block/text.tsx

```diff
@@ -843,13 +850,12 @@ const BlockText = observer(class BlockText extends React.Component<Props> {
 			e.preventDefault();
 			this.onSmile();
 		});
-
-		if (range && (range.from != range.to) && twineOpen.includes(key)) {
+		if (range && ((range.from != range.to) || block.isTextCode()) && twinePairs[key]) {
```

Comment:
This will work with a closing symbol as well, `Object.keys(twinePairs).includes(key)` I meant

Reply:
How about `Reflect.has(twinePairs,key)`

Reply:
Ok, never used it though)